### PR TITLE
hemamanne adds hometown of Plano

### PR DESCRIPTION
**Pull request by hemamanne**
Hi all 

   I am proposing assing  Plano, TX to the list of Hometowns, but I have some questions:

1) If you haven't been to Plano before   -- What  are you looking forward to ?
2) If you have been  to Plano  - What did you enjoy the most? or least?
3)

### CODE REVIEW

File: hemamanne-plano.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Plano, TX
+
+### Resturants
```
Comment (michaelmsinclair) [inline hemamanne-plano.md]: Are there any Indian restaurants in Plano? 

If there are, are there ones worth visiting **OR** avoiding?

Comment (hemamanne) [inline hemamanne-plano.md]: > If there are, are there ones worth visiting OR avoiding?

worth:  Basera  (  Non -Hyderbadi Biryani  house 

> If there are, are there ones worth visiting OR avoiding?

Worth : basera ( Non- Vegeterian)

Avoid : Hyderabadi Biryani House

**Comment by hemamanne** [comment]
@DanSewell  
Added the lake attraction information

**Comment by michaelmsinclair** [comment]
@hemamanne :shipit:, let's get these attactions in and work on adding restaurants.

**Pull request merged**